ui/changes/src/index.js

```diff
@@ -233,6 +237,35 @@ function addRow(bugSummary) {
     risk_text.style.color = HIGH_RISK_COLOR;
     risk_text.textContent = 'Higher';
   }
+}
+
+async function populateVersions() {
+  var versionSelector = document.getElementById("releaseVersion")
+
+  let data = await landingsData;
+
+  var allVersions = new Set();
+    for (let date in data) {
+      for (let bug of data[date]) {
+        if (bug.versions.length) {
+          allVersions.add(...bug.versions)
+        }
+      }
+    }
+  var versions = [...allVersions]
+  versions.sort()
+
+  for (let version of versions) {
+    let el = document.createElement("option")
+    el.setAttribute("value", version)
+    el.textContent = version
+    el.selected = true
+    versionSelector.appendChild(el)
+  }
+}
+
+function renderTestingSummary(bugSummaries) {
+  let metaBugID = getOption("metaBugID");
 
   risk_column.append(risk_text);
 }
```

Comment:
There is a problem here, maybe a bad rebase?

Reply:
Seems so, will fix.